src/EditorFeatures/Core/Implementation/Commands/CommandHandlerService.cs

```diff
@@ -98,11 +98,7 @@ void ICommandHandlerService.Execute<T>(IContentType contentType, T args, Action
             }
             else
             {
-                // If there aren't any command handlers, just invoke the last handler (if there is one).
-                if (lastHandler != null)
-                {
-                    lastHandler();
-                }
+                lastHandler?.Invoke();
```

Comment:
Ah I didn't realize this was a result of running his fix against our code base.  Like this change even more now.